### DISCUSSION TITLE
Can not use ibus to write unicode character on main kitty window

### DISCUSSION THREAD

**Discussion by sytranvn** [kovidgoyal/kitty#6468] [Q&A] [answered]
I have set `GLFW_IM_MODULE` in `kitty.conf` but I cannot type in other language.
```
env GLFW_IM_MODULE=ibus
```
But if I open a different `kitty` window from current one then it work.

![Screenshot from 2023-07-15 14-51-00](https://github.com/kovidgoyal/kitty/assets/13009812/e7d0083e-38f6-4c26-b54c-bdef7af4b6a9)

In main window, typing `aa` or `af` with ibus should become `â` and `à`.
What can be wrong with my default kitty window?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/faq/#things-behave-differently-when-running-kitty-from-system-launcher-vs-from-another-terminal

On Sat, Jul 15, 2023 at 12:52:01AM -0700, Sy Tran Dung wrote:
> I have set `GLFW_IM_MODULE` in `kitty.conf` but I cannot type in other language.
> ```
> env GLFW_IM_MODULE=ibus
> ```
> But if I open a different `kitty` window from current one then it work.
> 
> ![Screenshot from 2023-07-15 14-51-00](https://github.com/kovidgoyal/kitty/assets/13009812/e7d0083e-38f6-4c26-b54c-bdef7af4b6a9)
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6468
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by sytranvn** [reply]
`GLFW_IM_MODULE` was not available when run as desktop launcher. So I added it to  `.gnomerc`
